docker/windows_ci_vs_2019

```diff
@@ -4,12 +4,14 @@ FROM windows_ci_vs_base
 
 COPY `
   scripts/windows_install_vs_2019.ps1 `
+  scripts/windows_install_deps.ps1 `
```

Comment:
I think this should be in the windows_ci_vs_base layer. That way we can test both vs_2019 and vs_2021 eventually, with the tools available.